connectorx/src/transports/postgres_arrow.rs

```diff
@@ -48,6 +46,24 @@ impl<'r> TypeConversion<&'r str, String> for PostgresArrowTransport {
     }
 }
 
+impl<'r> TypeConversion<&'r str, &'r str> for PostgresArrowTransport {
```

Comment:
Following the comment above, I think this impl can be replaced by an `all` rule.